authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -0,0 +1,734 @@
+                    "kms:CreateGrant",
+                    "kms:ListGrants"
+                ],
+                "Resource": "*",
+                "Condition": {
+                    "Bool": {
+                        "kms:GrantIsForAWSResource": true
+                    }
+                }
+            }
+        ]
+    }
+}
+----
+. Prepare to run the {product-title} installer:
+
+.. Create the `install-config.yaml` file:
++
+[source,terminal]
+----
+$ ./openshift-install create install-config
+----
+
+.. Configure the cluster to install with the CCO in manual mode:
++
+[source,terminal]
+----
+$ echo "credentialsMode: Manual" >> install-config.yaml
+----
+
+.. Create install manifests:
++
+[source,terminal]
+----
+$ ./openshift-install create manifests
+----
+
+.. Create a `tls` directory, and copy the private key generated previously there:
++
+[NOTE]
+====
+The target file name is important.
+====
++
+[source,terminal]
+----
+$ mkdir tls ; cp <path-to-service-account-signer> ./tls/bound-service-account-signing-key.key
+----
+
+.. Create a custom `Authentication.config.openshift.io` CR with the file name `cluster-authentication-02-config.yaml`:
++
+[source,terminal]
+----
+$ cat << EOF > manifests/cluster-authentication-02-config.yaml
+iVersion: config.openshift.io/v1
+kind: Authentication
+metadata:
+  name: cluster
+spec:
+  serviceAccountIssuer: $OPENID_BUCKET_URL
+EOF
+----
+
+.. For each `CredentialsRequest` CR that is extracted from the release image, create a secret with the target namespace and target name that is indicated in each `CredentialsRequest`, substituting the AWS IAM role ARN created previously for each component:
++
+.Example secret file for `openshift-machine-api`:
++
+[source,terminal]
+----
+$ cat manifests/openshift-machine-api-aws-cloud-credentials-credentials.yaml
+apiVersion: v1
+stringData:
+  credentials: |-
+    [default]
+    role_arn = arn:aws:iam::123456789012:role/openshift-machine-api-aws-cloud-credentials
+    web_identity_token_file = /var/run/secrets/openshift/serviceaccount/token
+kind: Secret
+metadata:
+  name: aws-cloud-credentials
+  namespace: openshift-machine-api
+type: Opaque
+----
+
+[id="sts-mode-installing-manual-run-installer"]
+==== Running the installer
+
+. Run the {product-title} installer:
++
+[source,terminal]
+----
+$ ./openshift-install create cluster
+----
+
+. Continue to xref:#sts-mode-installing-verifying[Verifying the installation].
+
+[id="sts-mode-installing-verifying"]
+=== Verifying the installation
+
+. Connect to the {product-title} cluster.
+
+. Verify that the cluster does not have `root` credentials:
++
+[source,terminal]
+----
+$ oc get secrets -n kube-system aws-creds
+----
++
+The output should look similar to:
++
+[source,terminal]
+----
+Error from server (NotFound): secrets "aws-creds" not found
+----
+
+. Verify that the components are assuming the IAM roles that are specified in the secret manifests, instead of using credentials that are minted by the CCO:
```

Comment:
What is a _secret manifest_? Seems to be the first reference to that term.

Reply:
afaict, yes. I think manifest is probably a better term?